.github/workflows/docs.yml

```diff
@@ -20,13 +20,15 @@ jobs:
     - name: Install dependencies
       working-directory: l5kit
       run: |
-        python -m pip install --upgrade pip
-        pip install -r requirements.txt
+        pip install --upgrade pip
+        pip install pipenv
+        pipenv install --dev
```

Comment:
Always use `pipenv sync` to make sure that you install packages from the lock file.
No need for `--dev` since you are not developing here

Reply:
Currently our doc tools are installed as `dev` dependencies so they need `--dev`, but maybe that's not a best practice?

Reply:
docs are not part of dev :)
Dev packages are packages needed for development and documentation is not part of that

Reply:
I agree, but they are also not part of the library itself right? Should they be install together with it regardless?

Reply:
the dev packages should be installed when you want to do development :) In CI there is no development so there should be no --dev. Whatever is needed to produce some artifacts or to test something should not be dev